README.md

```diff
@@ -5,12 +5,10 @@
 [![License](https://img.shields.io/cocoapods/l/FirebaseHelper.svg?style=flat)](http://cocoapods.org/pods/FirebaseHelper)
 [![Platform](https://img.shields.io/cocoapods/p/FirebaseHelper.svg?style=flat)](http://cocoapods.org/pods/FirebaseHelper)
 
```

Comment:
Here, I suggest adding a brief introduction (and maybe some bullets) about what exactly FirebaseHelper is, and what it can do. Maybe something like: 

> FirebaseHelper is a small wrapper over Firebase's realtime database, providing streamlined methods to get, set, delete, and increment values.
>
> ## Features
> - [x] Setup Firebase Realtime Database Ref
> - [x] Read values (get)
> - [x] Write/Update values (set)
> - [x] Remove values (delete)
> - [x] Increment values (increment)
> - [ ] -insert future functionality here-

Reply:
Good idea! I'll work on this.